packages/reporter-vue/src/header/ReporterHeader.spec.js

```diff
@@ -1,48 +1,53 @@
 import ReporterHeader from './ReporterHeader.vue'
-import { useStatsStore } from '../store/reporter-store'
-// import { mount } from '@cypress/vue'
 import { h } from 'vue'
-import VTooltip from 'v-tooltip'
-
-const style = {
-  resize: 'both',
-  overflow: 'auto',
-  display: 'block',
-  color: 'black',
-  background: 'rgba(100,100,100, 0.1)'
-}
-
-it.only('renders', () => {
-  const timeout = 1512
-  const wrapper = {
-    setup() {
-      // const store = useStatsStore()
-      // // const reporter = useReporterStore()
+import { useStatsStore } from '../store/reporter-store'
```

Comment:
Could we use TS for our test files? I'd rather have `.ts` files, even if we need a handful of `ts-ignore`, although not sure we even need any here if we have the correct `.vue` shim.